tests/build_tests/super_errors/expected/jsx_type_mismatch_float.res.expected

```diff
@@ -8,6 +8,7 @@
   18 [2m│[0m 
 
   This has type: [1;31mfloat[0m
-  But it's expected to have type: [1;33mReact.element[0m [2m(defined as[0m [1;33mJsx.element[0m[2m)[0m
+  But children of JSX fragments are expected to have type:
+    [1;33mReact.element[0m [2m(defined as[0m [1;33mJsx.element[0m[2m)[0m
```

Comment:
it's less true here since fragments only accept `React.element` children that can indeed be plural.